src/crypto-api.ts

```diff
@@ -479,6 +480,17 @@ export enum CrossSigningKey {
     UserSigning = "user_signing",
 }
 
+/**
+ * Cross signing key
+ * More information https://spec.matrix.org/v1.7/client-server-api/#post_matrixclientv3keysdevice_signingupload
```

Comment:
![image](https://github.com/matrix-org/matrix-js-sdk/assets/1389908/eaa97e15-086e-45cb-865f-75cea9d89248)


```suggestion
 *
 * @see https://spec.matrix.org/v1.7/client-server-api/#post_matrixclientv3keysdevice_signingupload
```